unstructured/partition/html/partition.py

```diff
@@ -151,7 +157,7 @@ def encoding(self) -> str | None:
         return self._encoding
 
     @lazyproperty
-    def html_str(self) -> str:
+    def html_text(self) -> str:
```

Comment:
Is this a public attribute that we need to tell users is changing?

Reply:
No. `HtmlPartitionerOptions` hasn't been published or documented, so there's no contract as to its behavior. There still isn't actually. It's theoretically public but I wouldn't say it's part of the interface. As long as `partition_html()` fulfills its contract, `HtmlPartitionerOptions` is an implementation detail. Also, I can't think of what use it would be to anyone just now :)